### PR TITLE
docs: several enhancements on docs including index and installation

### PR DESCRIPTION
**Pull request by d0choa**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "gentropy"
 # !! version is managed by semantic_release
 version = "0.0.0"
-description = "Open targets Genetics Portal Python ETL"
+description = "Open Targets python framework for post-GWAS analysis"
 authors = ["Open Targets core team"]
```
Comment (DSuveges) [inline pyproject.toml:6]: Is the genetics team part of the "core team"? If not, we should also mention them.

**Review by DSuveges — APPROVED** [review]
I had only one quesiton, which is also not tightly related to this PR.

File: pyproject.toml (hunk 1 above)
Comment (d0choa) [reply pyproject.toml:6]: they are

**Comment by d0choa** [comment]
there is another branch from @addramir. I'll consolidate before merging

**Pull request merged**